docs/Running-Tachyon-Fault-Tolerant-on-EC2.md

```diff
@@ -4,4 +4,143 @@ title: Running Tachyon with Fault Tolerance on EC2
 nickname: Tachyon on EC2 with Fault Tolerance
 group: User Guide
 priority: 3
----
\ No newline at end of file
+---
+
+Tachyon with Fault Tolerance can be deployed on EC2 using the [Vagrant scripts](https://github.com/amplab/tachyon/tree/master/deploy/vagrant) that come with Tachyon. The scripts let you create, configure and destroy clusters that come automatically configured with HDFS.
+
+# Prerequisites
+
+**Install Vagrant and the AWS plugins**
+
+Download [Vagrant](https://www.vagrantup.com/downloads.html)
+
+Install AWS Vagrant plugin:
+
+    vagrant plugin install vagrant-aws
+    vagrant box add dummy https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box
+
+**Install Tachyon**
+
+Download Tachyon to your local machine, and unzip it:
+
+    $ wget https://github.com/amplab/tachyon/releases/download/v{{site.TACHYON_RELEASED_VERSION}}/tachyon-{{site.TACHYON_RELEASED_VERSION}}-bin.tar.gz
```

Comment:
I think going forward, we should use the tachyon-project domain for the wget downloads:

http://tachyon-project.org/downloads/files/0.7.1/tachyon-0.7.1-bin.tar.gz